pkgs/applications/science/machine-learning/starspace/default.nix

```diff
@@ -0,0 +1,29 @@
+{lib, stdenv, fetchFromGitHub, boost165}:
+
+stdenv.mkDerivation rec {
+  pname = "starspace";
+  version = "8aee0a950aa607c023e5c91cff518bec335b5df5";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "starspace";
+    rev = version;
```

Comment:
```suggestion
    inherit rev;
```

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/109651/commits/f6b9aa723fcf68ccd270b5278ceeacd3e431afc3